tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2976,11 +2923,8 @@ tensorflow::Status ConvertSquare(OpConverterParams* params) {
 tensorflow::Status ConvertReduce(OpConverterParams* params) {
   const auto& inputs = params->inputs;
   const auto& node_def = params->node_def;
-  if (inputs.size() != 2 || !inputs.at(0).is_tensor() ||
-      !inputs.at(1).is_weights()) {
-    return tensorflow::errors::InvalidArgument(
-        "Input expects tensor and weights, at", node_def.name());
-  }
+  TF_RETURN_IF_ERROR(
+      CheckInputsWeights(params, {{"input", false}, {"axis", true}}));
```

Comment:
s/axis/reduction_indices?

Reply:
Looks like it is just named axis:
https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/mean
https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/max
https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/prod